sql/core/src/test/resources/sql-tests/inputs/columnresolution-views.sql

```diff
@@ -13,10 +13,8 @@ DROP VIEW view1;
 -- Test scenario with Global Temp view
 CREATE OR REPLACE GLOBAL TEMPORARY VIEW view1 as SELECT 1 as i1;
 SELECT * FROM global_temp.view1;
--- TODO: Support this scenario
 SELECT global_temp.view1.* FROM global_temp.view1;
```

Comment:
Thanks @gatorsmile and @cloud-fan.  

Hi @cloud-fan,  
The use-case is to support fully qualified column name in queries to remove ambiguity in the naming.  

For e.g. `select db2.t1.i1 from db1.t1, db2.t1 where db1.t1.i2 = db2.t1.i3`

Today Spark does not support this.   The workaround is to create alias for the table.
 
Yes, the lookup for the table is coming from the ‘from’ clause. At a high level, changes include keeping track of the qualifier(including the database name) when the lookup happens and enhancing the column resolution logic in the analyzer to resolve fully qualified column name from a logical plan node’s children.  
There are more details in the design doc attached to the[ SPARK-19602](https://issues.apache.org/jira/browse/SPARK-19602)

Thanks.